version: "3.9"

services:
  python36:
    container_name: cont-py38
    hostname: py38
    build:
      context: .
      dockerfile: Dockerfile
    # entrypoint: /usr/src/entrypoint.sh # da un error y tira el contenedor
    volumes:
      - $HOME/projects/prj_python/pyprint/pyprint:/usr/src/pyprint

    # debug vscode
    ports:
       - "8008:8008"
    # mantiene el contenedor en ejecución
    command: tail -f /dev/null

    networks:
      - pyprintnet

networks:
  pyprintnet:
      name: pyprintnet
